api/src/main/java/org/eclipse/microprofile/openapi/annotations/ExternalDocumentation.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright 2017 SmartBear Software
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.eclipse.microprofile.openapi.annotations;
```

Comment:
I would rename the package to be org.eclipse.microprofile.openapi instead. No need to add "annotations" suffix.

Reply:
Please add package-info.java to version the api package correctly.

Reply:
@Emily-Jiang - I named this `annotations` because we will have a `models` folder and something like a `apis` folder, so wanted to better encapsulate each one.

Agreed on the `package-info.java`, will add those.